app/src/main/java/com/mymeetings/pairpomodoro/view/MainActivity.kt

```diff
@@ -158,6 +170,14 @@ class MainActivity : AppCompatActivity(), ServiceConnection {
         bindService(Intent(this, PomodoroService::class.java), this, Context.BIND_AUTO_CREATE)
     }
 
+    override fun onResume() {
+        super.onResume()
+        timerProgressBar.max =
```

Comment:
Why this has to be executed in every onResume

Reply:
There's a bug otherwise and the progressbar max is reset to zero. But after replacing with Android progress bar it might have been fixed. I'll test this scenario.

Reply:
Yes. I got it. But we should get it from Timer as the TimerPreference value may vary when user go with sync mode.

Reply:
This issue is now fixed.